app/src/main/AndroidManifest.xml

```diff
@@ -18,7 +18,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".EventActivity">
+        <activity android:name=".SearchEventActivity">
```

Comment:
@ebruersoy since this is an example app, activity name should be eventactivity, not search related one.